clang/lib/Sema/SemaExpr.cpp

```diff
@@ -5570,8 +5570,10 @@ ExprResult Sema::BuildCXXDefaultArgExpr(SourceLocation CallLoc,
           /*SkipImmediateInvocations=*/NestedDefaultChecking))
     return ExprError();
 
+  Expr *RewrittenExpr = Init == Param->getDefaultArg() ? nullptr : Init;
```

Comment:
```suggestion
  Expr *RewrittenExpr = (Init == Param->getDefaultArg() ? nullptr : Init);
```

MINOR preference to reverse this condition?  But not sure enough to suggest this.

Reply:
Fixed